mlos_bench/mlos_bench/tests/launcher_parse_args_test.py

```diff
@@ -70,7 +77,9 @@ def test_launcher_args_parse_1(config_paths: List[str]) -> None:
     # Check overriding values in a file from the command line.
     assert launcher.global_config['test_global_value_2'] == 'from-args'
     # Check that we can expand a $var in a config file that references an environment variable.
-    assert path_join(launcher.global_config["varWithEnvVarRef"], abs_path=True) == path_join(os.getcwd(), "foo", abs_path=True)
+    assert path_join(launcher.global_config["pathVarWithEnvVarRef"], abs_path=True) \
+        == path_join(expected_root_pwd, "foo", abs_path=True)
```

Comment:
maybe
```suggestion
    assert os.path.samepath(path_join(launcher.global_config["pathVarWithEnvVarRef"]),
                            path_join(expected_root_pwd, "foo"))
```

Reply:
The call is `samefile` I think, not `samepath` and doesn't work for files that don't exist unfortunately.

```python
>>> os.path.samefile("foo", "./foo")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<frozen genericpath>", line 100, in samefile
FileNotFoundError: [Errno 2] No such file or directory: 'foo'
```